packages/common/src/common.ts

```diff
@@ -450,14 +452,14 @@ export class Common {
       if (!(eip in EIPs)) {
         throw new Error(`${eip} not supported`)
       }
-      const minHF = this.gteHardfork(EIPs[eip]['minimumHardfork'])
+      const minHF = this.gteHardfork((EIPs as any)[eip]['minimumHardfork'])
```

Comment:
Here and at some more places are some `any` typings left. This has the reasoning that we would otherwise have to hard-specify the EIP numbers again at the typing level, and this would eventually limit the expandability of the library.

At the same time I think typing has already significantly improved since the format now is already checked for the configuration objects themselves.

No strong opinion on this though.